SnakeGame/model.py

```diff
@@ -1,18 +1,6 @@
 import enum
 
-class GameModel(object):
-
-    class CellContent(enum.Enum):
```

Comment:
Why were these deleted?

Reply:
They weren't deleted. They were moved outside the GameModel class. The reason for that is, if we were to import those enums in the other files (as we will), the previous way would have been: `from model import GameModel`. We would have had to reference the enums as `GameModel.CellContent`. That would be cumbersome. 

The way it's done now would be like: `from model import CellContent`